docker/Dockerfile.arm64

```diff
@@ -1,5 +1,12 @@
 FROM arm64v8/debian
```

Comment:
This doesn't work this easily, because the build of the docker images is done on an amd64 system, thus, it cannot execute apt-get.

Reply:
Hmm ok, that seems like a misconfigured build system but I don't know enough Go to dig into it. Is there a bug tracking fixing this? It'd be great if the .arm* Dockerfiles (or some other documentation) had a comment noting that they are less functional than the default Dockerfile.

Reply:
Not directly, but there is #350. This problem is more or less unrelated to Go, there should be a way for emulating arm on a amd64 system, but this needs to be integrated into the build pipeline.

Reply:
Any update please ?
not sure what is the difficulty, but if `wget` is easier, then that should work as well